apis/apis.json

```diff
@@ -32,6 +34,8 @@
 
   {
     "id": "Google.Cloud.Datastore.V1",
+    "productName": "Google Cloud Datastore",
+    "productUrl": "https://cloud.google.com/datastore/docs/concepts/overview",
```

Comment:
Could do. I was trying to leave all the URLs as they were for now... (Having them in one place makes this sort of inconsistency much easier to spot :)